src/Services/UploadImageService.php

```diff
@@ -10,6 +10,8 @@
  */
 class UploadImageService extends AbstractUploadService
 {
+    const R_MD5_MATCH = '/\?v=[a-f0-9]{32}$/';
```

Comment:
maybe check `.jpg` too, with a regex like `\.jpg\?v=...` ? Is `v=...` mandadory?

Reply:
I don't think so, unless we'll always have jpg image in backpack ? Maybe I've misunderstood you :)

Reply:
@jguyomard is right, without this check you enlarge the support to all formats, see my comment below.